godtools/App/Services/Renderer/Views/MobileContent/Views/Pages/MobileContentPagesViewModel.swift

```diff
@@ -61,12 +63,31 @@ class MobileContentPagesViewModel: NSObject {
                 self?.updateTranslationsIfNeeded()
             }
             .store(in: &cancellables)
+        
+        setUpCountLanguageUsages()
     }
     
     deinit {
 
     }
     
+    private func setUpCountLanguageUsages() {
+        
+        currentPageRenderer.flatMap { currentPageRenderer -> AnyPublisher<UserCounterDomainModel, Error> in
+            
+            let currentLocaleId = currentPageRenderer.language.localeIdentifier
+            let locale = Locale(identifier: currentLocaleId)
+            
+            return self.incrementUserCounterUseCase.incrementUserCounter(for: .languageUsed(locale: locale))
```

Comment:
Hey @rachaelblue I like this idea here, the only problem is there are situations when a page renderer may re-trigger the same page.  So we would end up duplicating language counts here.

One case is when a tool is first launched I believe AutoLayout did finish layout constraints will re-trigger a page renderer so this would get captured twice when a tool opens.  There may be other cases that re-trigger a current page to render.

Instead of sinking you could go down below into the ```setPageRenderer(pageRenderer:``` method and check for a language change and if so trigger the counter.

```

func setPageRenderer(pageRenderer: MobileContentPageRenderer) {

let languageChanged: Bool = currentPageRenderer.value.language.localeIdentifier != pageRenderer.language.localeIdentifier
        
        if languageChanged {
            print("trigger language counter")
        }

....
```